### DISCUSSION TITLE
posts/birth/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#30] [Comments]
# life/birth/

我的时间观 6 月 12 是我的农历生日。 大学以前，每次生日都是大表姐亲自做或者买的蛋糕。 那是我快乐的童年时光。 在天津服役两年回到大学之后，我的年龄意

https://shitao5.org/life/birth/

**Comment by yihui** [comment]
生日快乐！你的农历生日和我就差一天。

**Reply by Shitao5** [reply]
生日快乐！同是狮子座😁

**Reply by yihui** [reply]
星座是按公历来的，我们公历生日差十几天，我是巨蟹，哈哈。

**Reply by Shitao5** [reply]
好嘛，又算术捉急了，抱歉抱歉。yihui 去年似乎过了一个农历和公历重合的生日。

**Reply by yihui** [reply]
嗯，这次你算对了！
